src/admin/python/lsst/qserv/admin/cli/script.py

```diff
@@ -706,7 +705,42 @@ def enter_replication_controller(
         print the command and arguments that would have been run.
     """
 
-    def set_initial_configuartion(workers: Sequence[str], xrootd_manager: str) -> None:
+    def get_worker_id(qserv_worker_db: str) -> str:
```

Comment:
It seems like this may be more generally useful, could probably be in the utils file (in the same dir as this file). Also, the multiple nested functions in entery_proxy seems like too much (the other one made sense to me because it was being used as a lambda, but maybe even it belongs somewhere else). But since getting the worker id may be useful other places, move it anyway... agree?

Reply:
Good suggestion! I will move it.

Reply:
Well, I have to correct myself - moving this method into a different script is not possible because its implementation depends on data members and other methods defined within the file `scipt.py`. So, I would just leave it here for now.